Packs/GoogleCloudSCC/README.md

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+<head>
+<h1>Google Cloud Security Command Center</h1>
+</head>
+<body>
+
+<~XSOAR>
+ <a href="https://xsoar.pan.dev/docs/reference/integrations/google-cloud-scc">Google Cloud Security Command Center XSOAR integration</a>
+</~XSOAR>
+
+
+<h2>
+Overview
+</h2>
+
+<p>
+Google Cloud Security Command Center is a platform that offers deep visibility into cloud infrastructure.
+<br>
+Google Cloud Security Command Center identifies security threats, and provides proactive measures to mitigate risks. 
+<br>
+It consolidates security-related data, offers real-time monitoring and alerts, enables continuous security assessment, and provides recommendations to improve cloud security posture.
+</p>
+
+<~XSIAM>
+
+<h2> What does this pack do?</h2>
+<p>
+Google Cloud Security Command Center pack help organizations to monitor, identify and prevent security events on Google Cloud Platform, detect vulnerabilities on Google Cloud environment and provides instructions and recommendations to improve cloud security.
+</p>
+<br>
+
+<h3> Logs normalization supports the following data:</h3>
+
+<strong>Finding</strong> — A record of a threat, vulnerability, or misconfiguration that a certain service has found in Google Cloud environment. 
+Findings show the issue that was detected, the resource that is affected by the issue, and guidance on how you can address the issue.
+<br>
+<br>
+
+<details>
+<summary> For <a href="https://cloud.google.com/logging/docs/audit">Google Cloud audit logs</a> normalization follow this procedure</summary>
+<p>
+
+1. Go to marketplace and search for <i><strong>Google Cloud Logging</i></strong>.<br>
+2. Install <i><strong>Google Cloud Logging</i></strong>.<br>
+3. Go to <i><strong>Data Sources</i></strong> and <i><strong>Add New Instance</i></strong><br>
+4. Connect <i><strong>Google Cloud Platform</i></strong> data source<br>
+5. Insert subscription name (Ingest logs and data from a GCP Pub/Sub section 3)<br>
+6. Insert credentials file (Ingest logs and data from a GCP Pub/Sub section 4)<br>
+7. Select <i><strong>Flow or Audit Logs</i></strong><br>
+8. Go to Query builder and use the dataset - google_cloud_logging_raw<br>
+
+</p>
+<h4> Note </h4>
+<p>
+To include audit logs related to Google Cloud Security Command Center only, you will need to add inclusion filter on the log router sink. <br>
+Add the filter <i><strong>protoPayload.serviceName="securitycenter.googleapis.com"</i></strong> <br>
+as describe in <a href="https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Ingest-logs-and-data-from-a-GCP-Pub/Sub?tocId=RyZP~~W~YWbOPGKAClIQHg">section 2.c</a>. <br>
+</p>
+
+<a href="https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Ingest-logs-and-data-from-a-GCP-Pub/Sub?tocId=RyZP~~W~YWbOPGKAClIQHg">Ingest logs and data from a GCP Pub/Sub</a>
```

Comment:
```suggestion
as describe in <a href="https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Ingest-logs-and-data-from-a-GCP-Pub/Sub?tocId=RyZP~~W~YWbOPGKAClIQHg">section 2.c</a> 
<a href="https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Ingest-logs-and-data-from-a-GCP-Pub/Sub?tocId=RyZP~~W~YWbOPGKAClIQHg">Ingest logs and data from a GCP Pub/Sub</a>.
</p>
```
Not sure what is section 2.c.  Since this step is in a dropdown section, I think you need to be more specific as to where it is or leave it out.  Not sure it adds any additional information.